---
layout: page
title: Resources
permalink: /resources/
---

## Curriculum vitae
<hr><hr>
A PDF copy of my up-to-date CV can be downloaded [here](https://github.com/lcwheeler/curriculum_vitae/blob/master/LucasCWheeler_CV.pdf). 


## Scientific software tools
<hr><hr>

#### Enzo

Enzo is a simple python library that allows mathematical models of metabolic pathways to be evolved between phenotypic states. I developed this approach for a [paper](https://www.biorxiv.org/content/early/2019/01/03/511089) on the simulated evolution of the anthocyanin biosynthesis pathway. Enzo is a wrapper for the [Tellurium](http://tellurium.analogmachine.org/) environment, which is an excellent tool for developing reproducible systems biology simulations in Python. Enzo is a work in progress and I am planning to add additional functionality as time goes by. The primary **purpose of its existence** is to allow the simulations performed in [Computational modeling of anthocyanin pathway evolution: Biases, hotspots, and trade-offs](https://www.biorxiv.org/content/early/2019/01/03/511089) and [Structure and contingency determine mutational hotspots for flower color evolution](https://www.biorxiv.org/content/10.1101/2020.08.18.256503v1) to be easily replicated. 

*To install or contribute to enzo, go to the [github repo](https://github.com/lcwheeler/enzo).*


#### Orbweaver

Orbweaver is a pet project that is still a work in progress. It is a Python library designed to analyze gene co-expression relationships. Currently the functionality is limited, but it contains some useful tools for calculating distance matrices and generating network data structures.

*To install or contribute to orbweaver, go to the [github repo](https://github.com/lcwheeler/orbweaver).*


#### Kmertime

Kmertime is a super simple tool, written in python and cython, for counting k-mers in sequence data. It is reasonably fast and uses a convenient "KmerSet" class for handling the data. Kmertime also has a built-in function for generating random sequences. 

*To install or contribute to kmertime, go to the [github repo](https://github.com/lcwheeler/kmertime).*


## Educational materials
<hr><hr>

#### Python for Scientists

Python is a very useful programming language that has become increasingly popular in the scientifc community in recent years. 
My friends and I found that there was a lack of a single centralized guide to help scientists set up a working Python environment. 
So, we decided to do something about it! We created a web page using [Read the Docs](https://readthedocs.org/) that walks through 
setting up a useful Python ecosystem and also contains brief overviews of popular scientific packages and visualization tools. 

*Follow this link to reach the guide:* 

[Python for Scientists](https://python-for-scientists.readthedocs.io/en/latest/#)


*To make a contribution to the guide:* 

Fork and clone the master [github repo](https://github.com/Zsailer/python-for-scientists)


#### Pyway
Emergent properties arise from interactions between individual agents within systems. Think of a school of fish or a flock of starlings. One of the most famous computational examples is the simulated world of Conway's game of life. By following just three simple rules that determine the behavior of individual agents in the model and their relationships to neighbors, beautifully complex and interesting patterns emerge. I created a simple Python package that contains all the tools necessary to simulate Conway's game of life on a grid of any size, using either random or user-defined initital conditions. Pyway is meant as a teaching tool. It simplifies the initialization of the game and allows visualizations to be made of game trajectories. 

*You can install pyway from it's [github repo](https://github.com/lcwheeler/pyway).* 


## Recorded presentations
<hr><hr>

[Colorado Native Plant Society 2023 in Denver, Colorado](https://www.youtube.com/watch?v=5jaUkolVVUI) 

[Solanaceae Seminar Series 2020, Online](https://www.youtube.com/watch?v=OhSREoCemU0&feature=youtu.be)

[Botany Virtual Conference 2020, Online](https://www.youtube.com/watch?v=FpXiIhmI6p8&t=21s)

[Evolution 2019 in Providence, Rhode Island](https://www.youtube.com/watch?v=py3jSv0BCgE)


## Datasets and article repos
<hr><hr>

[OSF repo for "Evolutionary walks through flower color space driven by gene expression in Petunia and allies (Petunieae)"](https://osf.io/zg9cu/)


[SRA BioProject RNA-seq dataset for "Transcription factors evolve faster than their structural gene targets in the flavonoid pigment pathway."](https://www.ncbi.nlm.nih.gov/sra/PRJNA746328)


[OSF repo for "Transcription factors evolve faster than their structural gene targets in the flavonoid pigment pathway."](https://osf.io/b7gcp/)


[Github repo for "Were ancestral proteins less-specific?"](https://github.com/harmslab/were-anc-less-specific)


[SRA BioProject phage display dataset used in "Were ancestral proteins less-specific?" and "Learning peptide recognition rules for a low-specificity protein."](https://www.ncbi.nlm.nih.gov/bioproject?LinkName=sra_bioproject&from_uid=11384227)


[OSF repo for "Structure and contingency determine mutational hotspots for flower color evolution."](https://osf.io/kxr23/)


[Zenodo repo for "Computational modeling of anthocyanin pathway evolution: Biases, hotspots, and trade-offs."](https://zenodo.org/record/2611739#.X9UWM8J7k8p)


[Github repo for "S100A5 binds Ca2+ and Cu2+ independently."](https://github.com/harmslab/wheeler-harms-S100A5-Ca-Cu-data)








