---
layout: post
title: New preprint -- Evolution of flower color genes in petunias and their wild relatives.
---


After several years of field trips to collect samples, raising plants in the greenhouse, preparing and sequencing RNA-seq libraries we have finally written our first manuscript on the phylo-transcriptomic analysis of the Petunieae clade of Solanaceae. It was a pleasure working with all of the co-authors on this project: Joseph Walker, Julienne Ng, Rocío Deanna, Amy Dunbar-Wallis, Alice Backes, Pedro Pezzi, Virginia Palchetti, Andrew Monaghan, Loreta Brandão de Freitas, Gloria Barboza, Edwige Moyroud, and Stacey Smith! 

In this study, we reconstruct the Petunieae (Solanaceae) species tree using a phylo-transcriptomic approach and study the molecular evolution of flower color genes. We find that transcription factors regulating the expression of flavonoid enzyme genes evolve more quickly than their targets. We also find little evidence to support the classic hypothesis that the position of enzymes in the pathway is tightly linked to their rate of evolution. Rather, the rate is linked to the expression level of the genes, which is also systematically lower for TFs. Finally, we find that the evolutionary rate of pathway-related genes is not strongly coupled to transitions in floral pigmentation. Instead, increases and losses of pigmentation appear to be associated with the presence/absence of additional Myb TF copies in certain lineages.

I am very grateful to everyone who worked on this manuscript! This is the result of an international collaboration with colleagues in Brazil, Argentina, and several other countries. Many people helped to conduct the field work, collecting samples in an array of amazing places where these beuatiful plants grow!


#### Manuscript Abstract

Evolutionary transitions in flower color often trace back to changes in the flavonoid biosynthetic pathway and its regulators. In angiosperms, this pathway produces a range of red, purple, and blue anthocyanin pigments. Transcription factor (TF) complexes involving members of the MYB, bHLH, and WD40 protein families control the expression of pathway enzymes. Here, we investigate flavonoid pathway evolution in the Petunieae clade of the tomato family (Solanaceae). Using transcriptomic data from 69 species of Petunieae, we estimated a new phylogeny for the clade. For the 65 species with floral transcriptomes, we retrieved transcripts encoding homologs of 18 enzymes and transcription factors to investigate patterns of evolution across genes and lineages. We found that TFs exhibit faster rates of molecular evolution than their targets, with the highly specialized MYB genes evolving fastest. Using the largest comparative dataset to date, we recovered little support for the hypothesis that upstream enzymes evolve slower than those occupying more downstream positions. However, expression levels inversely correlated with molecular evolutionary rates, while shifts in floral pigmentation were weakly related to changes affecting coding regions. Nevertheless, shifts in floral pigmentation and presence/absence patterns of MYB transcripts are strongly correlated. Intensely pigmented and patterned species express homologs of all three main MYB anthocyanin activators in petals, while pale or white species express few or none. Our findings reinforce the notion that regulators of the flavonoid pathway have a dynamic history, involving higher rates of molecular evolution than structural components, along with frequent changes in expression during color transitions.


You can read the preprint [here](https://ecoevorxiv.org/3tdfk)!

#### Figure 1 from the manuscript

![an image alt text]({{ site.baseurl }}/img/Fig1_flowers.png)

